Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;

```

Comment:
The padding value should include units (e.g., px) like 'padding: 10px;' for proper CSS syntax